packages/charts/src/components/ComposedChart/ComposedRechart.stories.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { ComposedRechart } from './ComposedRechart';
+import { Line } from 'recharts';
+import { Bar } from 'recharts';
+
+const dataset = [
+  {
+    xValue: 'January',
+    users: 100,
+    sessions: 300,
+    volume: 756
+  },
+  {
+    xValue: 'February',
+    users: 230,
+    sessions: 330,
+    volume: 880
+  },
+  {
+    xValue: 'March',
+    users: 240,
+    sessions: 404,
+    volume: 700
+  },
+  {
+    xValue: 'April',
+    users: 280,
+    sessions: 80,
+    volume: 604
+  },
+  {
+    xValue: 'May',
+    users: 100,
+    sessions: 300,
+    volume: 756
+  },
+  {
+    xValue: 'June',
+    users: 230,
+    sessions: 330,
+    volume: 880
+  },
+  {
+    xValue: 'July',
+    users: 20,
+    sessions: 470,
+    volume: 450
+  },
+  {
+    xValue: 'August',
+    users: 220,
+    sessions: 180,
+    volume: 104
+  },
+  {
+    xValue: 'September',
+    users: 200,
+    sessions: 360,
+    volume: 879
+  },
+  {
+    xValue: 'October',
+    users: 250,
+    sessions: 500,
+    volume: 200
+  },
+  {
+    xValue: 'November',
+    users: 240,
+    sessions: 404,
+    volume: 700
+  },
+  {
+    xValue: 'December',
+    users: 280,
+    sessions: 80,
+    volume: 604
+  }
+];
+
+export default {
+  title: 'Charts / ComposedChart',
+  component: ComposedRechart
+};
+
+const clickHandler = (data) => {
+  console.log(data);
+};
+
+const renderComposedChart = () => (
+  <ComposedRechart
+    width={'95%'}
+    height={'40vh'}
+    dataset={dataset}
+    labelKey={'xValue'}
+    legendClickHandler={clickHandler}
```

Comment:
please use storybook actions for event handlers
```suggestion
    legendClickHandler={action('legendClickHandler')}
```
import it via `import { action } from '@storybook/addon-actions';`